src/main/java/com/dsrm/dsrmbackend/tables/Room.java

```diff
@@ -0,0 +1,38 @@
+package com.dsrm.dsrmbackend.tables;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.Set;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "rooms")
+public class Room {
+    @Id
+    private Long Id;
+    @Column(name = "room_number")
+    private Integer roomNumber;
+    private Integer floor;
+    @OneToMany
+    @JoinColumn(name = "type_id")
+    private Set<RoomType> roomType;
+    @Column(name = "max_capacity")
+    private Integer maxCapacity;
+    @OneToMany
+    @JoinColumn(name = "key_owner_id")
+    @Column(name = "key_owner_id")
```

Comment:
@Column is not necessary when you use @JoinColumn and as above I mentioned it should be other way ManyToOne here